AssetManagementTestProject/TestSetup/NUnitWebTestSetup.cs

```diff
@@ -25,9 +30,14 @@ public void SetUp()
         Asserter = new Asserter.Asserter();
         MenuBarLeft = new LeftMenuPage();
         ChangePw1stTime = new ChangePassword1stTimePage();
+        ChangePassword = new ChangePassword();
+
+        AuthorizationService = new AuthorizationService();
+        Token = AuthorizationService.GetLoginToken(Constant.ADMIN_USERNAME_HN, Constant.BASE_ADMIN_PASSWORD);
+
```

Comment:
This will create a new user with API for EVERY TEST CASE, which means for test cases you test with a default account i.e. ADMIN_USERNAME_HN, there will be a new user created.
Consider moving this to the test cases which require creating new user to test only

Reply:
My mistake, you can now use this in the test setup + create new user after I updated the clean data function in the TearDown.
It could be like this:
![image](https://user-images.githubusercontent.com/79431746/206345980-fe2a14ce-b214-4a7b-a7dc-6189be775012.png)

Reply:
Thank you for your advise